Postgres 16 support

Tried building with latest postgres 16 but saw some build failures like
```
src/hll.c: In function ‘FunctionOid’:
src/hll.c:326:36: error: too few arguments to function ‘stringToQualifiedNameList’
  List *qualifiedFunctionNameList = stringToQualifiedNameList(qualifiedFunctionName);
                                    ^~~~~~~~~~~~~~~~~~~~~~~~~
In file included from src/hll.c:53:0:
server/utils/regproc.h:28:14: note: declared here
 extern List *stringToQualifiedNameList(const char *string, Node *escontext);
              ^~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: *** [src/hll.o] Error 1
make[1]: *** Waiting for unfinished jobs....
src/hll.c:326:83: error: too few arguments to function call, expected 2, have 1
        List *qualifiedFunctionNameList = stringToQualifiedNameList(qualifiedFunctionName);
                                          ~~~~~~~~~~~~~~~~~~~~~~~~~                      ^
server/utils/regproc.h:28:1: note: 'stringToQualifiedNameList' declared here
extern List *stringToQualifiedNameList(const char *string, Node *escontext);
^
1 error generated.
make[1]: *** [src/hll.bc] Error 1
```
I know 16 is not yet supported but may I know when we can expect a PG16 compatible version to be released? Thank you!!

--- a/src/hll.c
+++ b/src/hll.c
@@ -142,7 +142,9 @@ static void hll_aggregation_restriction_hook(PlannerInfo *root, UpperRelationKin
 #endif
 
 static void MaximizeCostOfHashAggregate(Path *path);
+#if PG_VERSION_NUM < 160000
 static Oid get_extension_schema(Oid ext_oid);
+#endif
 static Oid FunctionOid(const char *schemaName, const char *functionName, int argumentCount, bool missingOk);
 static void InitializeHllAggregateOids(void);
 static bool HllAggregateOid(Oid aggregateOid);
@@ -257,6 +259,8 @@ InitializeHllAggregateOids()
 	aggregateValuesInitialized = true;
 }
 
+/* PG16 ships its own get_extension_schema */
+#if PG_VERSION_NUM < 160000
 /*
  * get_extension_schema - given an extension OID, fetch its extnamespace
  * Returns InvalidOid if no such extension.
@@ -308,6 +312,7 @@ get_extension_schema(Oid ext_oid)
 
 	return result;
 }
+#endif
 
 /*
  * FunctionOid searches for a given function identified by schema, functionName
@@ -323,7 +328,11 @@ FunctionOid(const char *schemaName, const char *functionName, int argumentCount,
 	Oid functionOid = InvalidOid;
 
 	char *qualifiedFunctionName = quote_qualified_identifier(schemaName, functionName);
-	List *qualifiedFunctionNameList = stringToQualifiedNameList(qualifiedFunctionName);
+	List *qualifiedFunctionNameList = stringToQualifiedNameList(qualifiedFunctionName
+#if PG_VERSION_NUM >= 160000
+					, NULL
+#endif
+			);
 	List *argumentList = NIL;
 	const bool findVariadics = false;
 	const bool findDefaults = false;

